File: level 22/homework/hw4.py
# 4) შექმენით 3 ცრილი სამივე იქნება განსხვავებული მონაცემთა ტიპიები  | ბოლეანი არა საჭირო | და შექმენით ცარიელი ცხრილი სადაც იქნება დასაწყისში ინტეჯერების ცხრილი  სტრინგების ცხრილი და მერე ფლოათების ცხრილი

intg = [2, 40, 50 ,3 , 55]

flt = [1.5, 5.50 , 60.40 , 5.5 , 6.9]

strng = ["giorgi", "lasha", "beqa" ,"giorgi", "irakli"]

empty = []

for element in intg:
    if type(element) == int:
        empty.append(element)
    
for element in strng:
    if type(element) == str:
        empty.append(element)
    
for element in flt:
    if type(element) == float :
        empty.append(element)
print(empty)

File: level 71/homework/hw4.js
//  შექმენით სია სადაც იქნება user-ების სახელი და გვარი და ახალ სიაში დაამატეთ ამ იუსერების  ინიციალები სახელის და გვარის პირველი ასოები, მაგალითად: გიორგი ბიბილაშვილი -> გ.ბ 


let arr = ["giorgi giorgobiani","beqa gviniashvili","lasha sirbiladze"]

let empty = []

for(let i of arr){
    var parts = i.split(" ")

    empty.push(parts[0][0]+ "." + parts[0][0])
}
console.log(empty)